cluster/manifests/02-skipper-validation-webhook/02-hostname-credentials-controller.yaml

```
@@ -124,6 +124,10 @@ spec:
               [{"container": "controller", "parser": "keyValue"}]
             logging/destination: "{{ .Cluster.ConfigItems.log_destination_both }}"
         spec:
+          tolerations:
+          - key: dedicated
+            value: cluster-seed
+            effect: NoSchedule
```

```suggestion
          # {{- if eq .Cluster.Provider "zalando-eks"}}
          tolerations:
          - key: dedicated
            value: cluster-seed
            effect: NoSchedule
          # {{ end }}
```

Let's limit this to EKS clusters as the only ones with cluster-seed nodes.